app/jobs/socure_docv_results_job.rb

```diff
@@ -46,6 +46,16 @@ def log_verification_request(docv_result_response:, vendor_request_time_in_ms:)
       ).except(:attention_with_barcode, :selfie_live, :selfie_quality_good,
                :selfie_status).compact,
     )
+    analytics.idv_socure_document_request_submitted(
+      **docv_result_response.to_h.merge(
+        docv_transaction_token: document_capture_session.socure_docv_transaction_token,
+        submit_attempts: rate_limiter&.attempts,
+        remaining_submit_attempts: rate_limiter&.remaining_count,
+        vendor_request_time_in_ms:,
+        async:,
+      ).except(:attention_with_barcode, :selfie_live, :selfie_quality_good,
+               :selfie_status).compact,
+    )
```

Comment:
@AShukla-GSA - this appears to be re-logging the verificationData request again.  We are intending to log the DocumentRequests (See below)

1. https://github.com/18F/identity-idp/blob/0032b193cb716ab7a37ec2c00321200f4e4ec9ab/app/controllers/idv/socure/document_capture_controller.rb#L36
2. https://github.com/18F/identity-idp/blob/0032b193cb716ab7a37ec2c00321200f4e4ec9ab/app/controllers/idv/hybrid_mobile/socure/document_capture_controller.rb#L25